package.json

```diff
@@ -66,6 +66,7 @@
     "@angular/platform-browser-dynamic": "^4.0.1",
     "@angular/platform-server": "^4.0.1",
     "@angular/router": "^4.0.1",
+    "c3": "^0.4.15",
```

Comment:
Drive by observation, we might want to lock down this version number, just in case something magically changes and breaks everything 🤔  (also many of the other deps are locked down, but some are not? 🤔 🤔 )